### DISCUSSION TITLE
Will sandpack support is_sse type sandboxes?

### DISCUSSION THREAD

**Discussion by plondon** [codesandbox/sandpack#253] [Q&A] [answered]
For example if you try to build a custom sandpack with the [nextjs template](https://codesandbox.io/api/v1/sandboxes/affectionate-lucy-ibemn) the compiler will complain and won't try to kick off a local development enviornment.

Thoughts on this type of support? Or is that not in scope for now?

Thanks!

**Comment by DeMoorJasper** [comment, marked as answer]
The `sse` sandboxes are backed by a container running on a server, therefore we can't easily support this in the sandpack library as you'd probably have to host your own servers in that case or integrate completely with our apis, which likely requires logins and paying above certain limits.

supporting these in sandpack isn't on the roadmap at the moment, but we are planning to make the embeds more lightweight in the future so maybe codesandbox embeds are a good solution for now?

**Reply by plondon** [reply]
Thanks for the response @DeMoorJasper! I would definitely be happy to incur the cost of hosting the servers myself, if thats a possibility.

In the meantime I can explore the possibility of embeds.

**Reply by plondon** [reply]
After exploring embeds for a bit I'm not sure if it will work exactly for my purpose, _unless_ there is a way to update the files of the embed via some sort of postMessage? i.e something like stackblitz's [applyFsDiff](https://developer.stackblitz.com/docs/platform/javascript-sdk/#vmapplyfsdiffdiff)?

**Comment by gitcommitankit** [comment]
@plondon @DeMoorJasper 
hey can you please tell me how can i get a nextjs-typescrit template in <SandpackProvider> (or it isn't available for now)?
